khard/carddav_object.py

```diff
@@ -57,7 +57,6 @@ def convert_to_vcard(name, value, allowed_object_type):
     raise ValueError("Error: " + name +
                      " must be a string or a list with strings.")
 
-
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/) tells us to use two empty lines between classes.